guides/v2.1/rest/apis.md

```diff
@@ -5,7 +5,7 @@ title: REST APIs (Swagger)
 menu_title: REST APIs (Swagger)
 version: 2.1
 menu_order: 2
-externalURL: http://devdocs.magento.com/swagger/index_21.html
+externalURL: {{ site.baseurl }}/swagger/index_21.html
```

Comment:
@jeff-matthews This breaks the bundle serve for me. Can you double check?

Reply:
@hguthrie you're right, noticed it as well as I tested it again. Will create a PR to fix.